protos/sources/vega/snapshot/v1/snapshot.proto

```diff
@@ -446,7 +441,7 @@ message PriceMonitor {
   repeated PriceRangeCache price_range_cache = 9;
   int64 ref_price_cache_time = 10;
   repeated DecimalMap ref_price_cache = 11;
-  repeated CurrentPrice prices_now = 12;
+  repeated string prices_now = 12;
```

Comment:
both this and the other change above will not work. you can't change the types and/or names of existing fields or it won't be backward compatible. you can add a new field, migrate to it, then later we can mark old field as unused.

Reply:
Would it work with latest changes?

Reply:
it would but it adds a bit of confusion by the different naming between the proto layer and typed.